conf/cassandra.yaml

```diff
@@ -951,3 +953,8 @@ windows_timer_interval: 1
 
 # Do not try to coalesce messages if we already got that many messages. This should be more than 2 and less than 128.
 # otc_coalescing_enough_coalesced_messages: 8
+data_file_directories:
+    - /Users/ltodorov/Desktop/Dev/palandra/data/data
+persistent_settings_directory: /Users/ltodorov/Desktop/Dev/palandra/data/psd
+commitlog_directory: /Users/ltodorov/Desktop/Dev/palandra/data/commitlog
+saved_caches_directory: /Users/ltodorov/Desktop/Dev/palandra/data/saved_caches
```

Comment:
Do we need those? In general, can you explain the changes to the cassandra.yaml file?

Reply:
We don't, these are changes to get things running locally, it was on oversight pushing the configs. I'll remove with a force push.